src/debug/inc/arm/primitives.h

```diff
@@ -30,7 +30,11 @@ typedef DPTR(CORDB_ADDRESS_TYPE)    PTR_CORDB_ADDRESS_TYPE;
 #define STACKWALK_CONTROLPC_ADJUST_OFFSET 2
 
 #define CORDbg_BREAK_INSTRUCTION_SIZE 2
+#ifdef LINUX32
```

Comment:
The break instruction code `0xde01` is hardcoded in Linux kernel. It's different from ARM spec and as I know both macOS (iOS) and Windows on ARM are using `0xdefe` (as in ARM spec). So this is Linux-specific.
Next, return address from SIGTRAP is different in ARM Linux and in x86 Linux, so the change of PC value should also happen for ARM Linux only.

Thanks for suggestion about `LINUX32` and `__linux__`.